src/pages/ProductList/ProductList.jsx

```diff
@@ -1,12 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './ProductList.scss';
+import Product from './component/Product';
+import Filter from './component/Filter';
 
 const ProductList = () => {
+  // 데이터 오는 곳
+  const [isSort, setIsSort] = useState(false);
+
   return (
     <div className="product-list">
-      <div className="container">ProductList</div>
+      <header className="list-header">
+        <h1 className="continent">아시아</h1>
+        <button
+          className="sort-button"
+          onClick={() => setIsSort(prev => !prev)}
+        >
+          <span className="sort-by">정렬 기준</span>
+          {isSort ? (
+            <img
+              className="sort-img"
+              alt="더보기"
+              src="images/icon/angle-up-solid.svg"
+            />
+          ) : (
+            <img
+              className="sort-img"
+              alt="더보기"
+              src="images/icon/angle-down-solid.svg"
+            />
+          )}
```

Comment:
```suggestion

            <img  
            className="sort-img" 
            alt="더보기" 
            src={`images/icon/${isSort?"angle-up-solid":"angle-down-solid"}.svg`}/>
          
```
조건에 따라 변하는 부분만 템플릿 리터럴로 처리해줄 수 있습니다.